parm/fv3lam_rrfs.xml

```diff
@@ -399,8 +399,8 @@
        </param>
 
        <param>
-       <shortname>SOILM_ON_DEPTH_BEL_LAND_SFC</shortname>
-       <pname>SOILMOI</pname>
+       <shortname>CISOILM_ON_DEPTH_BEL_LAND_SFC</shortname>
```

Comment:
@EricJames-NOAA The total soil moisture is generated as RRFS product as:
```
790:5274112429:d=2024022718:CISOILM:0-0 m below ground
``` 
There might be two soil level enters needed. 
The GFS is labeling as "0-2m" so enters in GFS xml are:
```
       <scale_fact_fixed_sfc1>2</scale_fact_fixed_sfc1>
       <level>0.</level>
       <scale_fact_fixed_sfc2>2</scale_fact_fixed_sfc2>
       <level2>200.</level2>
```

Reply:
@WenMeng-NOAA I spoke with @AndrewBenjamin-NOAA about this.  This field is really a column integrated field, but there is no way that I know of to indicate "entire soil column" or something like that.  

I am checking if I can just use one level to indicate simply "CISOILM:0 m underground" similar to the RAP SOILM fieldfield.

Reply:
@WenMeng-NOAA I confirmed I can output it as CISOILM:0 m underground

Is this approach OK with you?

Reply:
@EricJames-NOAA Your approach works to me since GFS and RRFS use different LSMs.